Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py

```diff
@@ -793,7 +793,9 @@ def _setup_cmdline_args(self, cmdline_args, executable, test_spec_list, workspac
         if type(executable) in [WinEditor, LinuxEditor]:  # Handle Editor CLI args since we need workspace context to populate them.
             test_cmdline_args += [
                 "--regset=/Amazon/Preferences/EnablePrefabSystem=true",
-                f"--regset-file={os.path.join(workspace.paths.engine_root(), 'Registry', 'prefab.test.setreg')}"]
+                f"--regset-file={os.path.join(workspace.paths.engine_root(), 'Registry', 'prefab.test.setreg')}",
+                "--log_IncludeTime=1"
+            ]
```

Comment:
Thanks for moving this `]` to a newline so we can easily add new arguments in the future :)